doc/RELEASE_NOTES.md

```diff
@@ -160,16 +158,16 @@ Finally, the Pbench Agent Ansible Galaxy collection provides roles for manipulat
 
 ## Removal of Gratuitous Software Installation
 
-The software required to run a particular benchmark, or a particular tool, is no longer gratuitously installed during the execution of a benchmark, or during tool registration.  If a benchmark convenience script or tool requires a certain version of software to be present, those checks will be performed and reported to the user as an error if the requirements are not met.
+The software required to run a particular benchmark workload, or a particular tool, is no longer gratuitously installed during the execution of the workload, or during tool registration.  If a benchmark convenience script or tool requires a certain version of software to be present, those checks will be performed and reported to the user as an error if the requirements are not met.
```

Comment:
I think my eyes always skipped over this before; but is "gratuitous" really the word we want here? While not literally incorrect, the word also has implications of "unnecessary" or extraneous ... and it's not, really; we just don't want Pbench as the intermediary orchestrator to be responsible for it.